Assets/MixedRealityToolkit/Interfaces/Diagnostics/IMixedRealityDiagnosticsSystem.cs

```diff
@@ -23,6 +23,16 @@ public interface IMixedRealityDiagnosticsSystem : IMixedRealityEventSystem, IMix
         /// </summary>
         bool ShowProfiler { get; set; }
 
+        /// <summary>
```

Comment:
@wiwei  FYI this is a breaking change technically, since anybody implementing the MRDiagnosticsSystem would now need to implement these two properties. I think it's fine, but we should add it to the release notes.

Reply:
Nice catch, thank you Julia!